extractors/bookseries/karelians/extractors/forest_extractor.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+import core.utils.regex_utils as regexUtils
+from core.pipeline_construction.base_extractor import BaseExtractor
+
+
+class ForestExtractor(BaseExtractor):
+    extraction_key = 'metsää'

```

Comment:
We should stick to English with all of the data output by `kaira` and also in the code itself (eg. names of extractors). Also, most of them are already in English, and maintain consistency is nice. Translating some of the existing keys to English would be nice.

Maybe something like `forestArea` or `forestAreaOwned` or something.. Also there's a farm area extractor, maybe check which key that uses. Would be nice to maintain consistency between the extraction keys :)